examples/nextjs/app-router/app/ServerIdentity.tsx

```diff
@@ -17,7 +17,7 @@ export const ServerIdentity = async function () {
             </span>
             <b>Server Identity</b>
             <span data-testid={'server-user-id'}>
-                {getUserIdentity()?.user_id}
+                {(await getUserIdentity()).user_id}
```

Comment:
this isn't really the intention of this line. It was meant to reflect what the user identity the SDK currently knows it has. It's calling a method that's still exported from the SDK, but actually wouldn't work if it was called so maybe we should remove it or make it work

Reply:
I could set the `user_id` as a global in the `devcycle.tsx` file, but I couldn't see a way to get the user_id from the context that is returned.

Reply:
yeah there currently isn't a way, we'd have to return the `getUserIdentity` function from here

Reply:
why I can't I just return it from the `devcycle.tsx` file like it is right now?

Reply:
you can but that wasn't the point of what it was doing before. It was printing what the SDK knows the user's identity to be

Reply:
well originally it was saying "here is what the user's identity is currently set to within the SDK" and now its saying "here is what I'm telling the SDK the user should be set to". The result is the same but the place where you're getting the info from is different. However I think we should just remove the ability to `getUserIdentity` from the SDK for now, so this is probably fine